benchmark/probe_snn_compile_boundary.py

```
@@ -0,0 +1,435 @@
+                str(Path(__file__).resolve()),
+                "--child",
+                "--case",
+                case,
+                "--phase",
+                phase,
+                "--seed",
+                str(args.seed),
+                "--output",
+                str(child_output),
+            ]
+            completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)